src/jlapi.c

```diff
@@ -576,16 +576,17 @@ static NOINLINE int true_main(int argc, char *argv[])
 
     if (start_client) {
         jl_task_t *ct = jl_current_task;
+        int ret = 1;
         JL_TRY {
             size_t last_age = ct->world_age;
             ct->world_age = jl_get_world_counter();
-            jl_apply(&start_client, 1);
+            ret = jl_unbox_int32(jl_apply(&start_client, 1));
```

Comment:
This should probably have a type-assert here first

Reply:
`_start` is internal, so I though it was fine, but might as well be defensive about it.